server/controllers/usersController.js

```diff
@@ -0,0 +1,23 @@
+import jwt from "jsonwebtoken"
+import userService from "../services/user.js";
+
+export function generateToken(req, res) {
+  res.send(process.env.TOKEN_SECRET);
+}
+
+export async function loginUser(req, res) {
+  const { username, password } = req.body;
+  const user = await userService.getUser(username);
+
+  if (user.name != username) {
+    return res.status(403).json({
```

Comment:
install http-status-codes
```suggestion
    return res.status(StatusCodes.FORBIDDEN).json({
```